lib/model/migrations/20230419-01-optimize-indices-sub-defs.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  // removing redundant indices from submissions
+  await db.raw('DROP INDEX public.submissions_draft_index;');
+  await db.raw('DROP INDEX public.submissions_formid_index;');
+  await db.raw('DROP INDEX public.submissions_formid_instanceid_index;');
+  await db.raw('DROP INDEX public.submissions_formid_createdat_index;');
+  // because we sort by createdAt, id for get and exports
+  await db.raw('CREATE INDEX submissions_formid_createdat_id_index ON public.submissions USING btree ("formId", "createdAt", id);');
```

Comment:
Did you carefully consider this order? Would be helpful to get your reasoning so I can check it rather than building it up myself against the different queries.